src/sites/link/linkdrop.net.js

```diff
@@ -515,6 +486,10 @@
 
   function getURLFromJQueryForm (jForm) {
     return new Promise((resolve, reject) => {
+      if (jForm.is('a') && jForm.attr('href')) {
```

Comment:
I added linksht, but the continue link after the countdown had no form, only a link. I could extract it into another class, but maybe it's useful here for other sites that copy this method